src/components/ContentSection.tsx

```diff
@@ -1,6 +1,6 @@
 import { FunctionComponent, ReactNode } from 'react'
 
-const COLORS = {
+export const COLORS = {
```

Comment:
Are you importing these colours somewhere? Just wondering what the need for the export is?

Reply:
`COLORS` was used to type the `color` prop in the FeatureWalkthrough component. It is no longer needed there so we can remove the export.